### DISCUSSION TITLE
Import / Exportation données

### DISCUSSION THREAD

**Discussion by victorgyt** [dbarzin/mercator#574] [Q&A] [answered]
Bonjour,
J'ai crée deux Mercator, l'un est plus avancé en terme de données que l'autre. 

Le nouveau que j'ai crée est maintenant sécurisé (mise en place du certificat HTTPS...) mais vide en terme de données, j'aimerais savoir s'il existe un moyen d'exporter manuellement depuis l'interface graphique les données pour les insérer dans le nouveau Mercator. 

- Savez vous s'il est possible de réaliser cette manipulation (Export/Import) depuis l'interface graphique ?

Merci ! 

V.G

**Comment by dbarzin** [comment]
Il n'existe pas de procédure permettant l'import/export de l'ensemble de la base de données depuis l'interface graphique.
C'est cependant très simple à faire en ligne de commande avec mysqldump :

    sudo mysqldump mercator1 > backup.sql
    sudo mysql mercator2 < backup.sql

Il faudra peut-être appliquer les migrations sur mercator2 si les deux projets ne sont pas au même niveau :

    php artisan migrate

**Comment by victorgyt** [comment, marked as answer]
Très bien merci.

**Comment by dbarzin** [comment]
Vous pouvez désormais importer des données à partir d’une feuille de calcul Excel directement depuis le menu Configuration → Import !
Un moyen simple et rapide pour initialiser ou mettre à jour vos référentiels ✨
